source/operators/rekognition/check_text_detection_status.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+###############################################################################
+# PURPOSE:
+#   Lambda function to check the status of a Rekognition job and save that job's
+#   data to the MIE dataplane when the job is complete.
+###############################################################################
+
+import os
+import boto3
+import json
+from botocore import config
+from MediaInsightsEngineLambdaHelper import OutputHelper
+from MediaInsightsEngineLambdaHelper import MasExecutionError
+from MediaInsightsEngineLambdaHelper import DataPlane
+
+operator_name = os.environ['OPERATOR_NAME']
+output_object = OutputHelper(operator_name)
```

Comment:
Isn't OutputHelper being deprecated?  i.e. use OperationHelper.  Been a while so maybe I'm wrong.

Reply:
That's a good question - I've simply copied the code from existing rek operators, then modified to fit the text detection method, so I'm unsure if there is a specific reason for that. @ianwow Do you have any insight on this?

Reply:
We have an issue for this, https://github.com/awslabs/aws-media-insights-engine/issues/99 but it hasn't been promoted in our sprint planning yet so I haven't done anything for it.

Reply:
Got it, makes sense. Fairly low priority imo, so I think we're fine for now since it isn't blocking anything or adding functionality.

Reply:
Yeah, it hasn't bubbled up to the top yet.  As I recall, the error handling doesn't work properly for operators that use the output helper, so it is introducing the same defect into the new operator.  I guess this one can be fixed with the rest.